content/livesync/connector/database.textile

```diff
@@ -0,0 +1,198 @@
+
+The Database Connector automatically deletes records from the table when records are successfully processed.
+
+<aside data-type='note'>
+<p>The Database Connector will always keep at least one outbox record in the table, specifically the one with the largest @sequence_id@. This record is still marked as processed via a tombstone and will be deleted when subsequent records are written to the outbox. This approach ensures that the backend endpoint called by your "sync function":/livesync/models#sync can always obtain the @sequence_id@ at the time that the state is read from the database.</p>
+</aside>
+
+h3(#why). Why use an outbox pattern?
+
+Clients using the "Models SDK":/livesync/models observe changes to the database state by consuming a stream of ordered database change events. An update to the data in the database must happen atomically, with the corresponding change event broadcast to all clients, so that they see a consistent view of the data.
+
+The Database Connector uses an outbox table to capture change events like this. When you update data in the database, you write a change event record to the outbox in the same database transaction as the update. This process ensures that change event records are written atomically to the outbox with the underlying update in the correct order.
+
+The Database Connector automatically retries failed publishes and only marks records as processed in the outbox after successfully publishing them. It uses idempotent publishing to ensure that messages are delivered without duplication, exactly-once.
+
+An additional benefit of the outbox pattern is that it decouples the underlying database schema from the change event schema published over Ably. This separation lets you adapt your database schema and data models without altering the code of your subscribing applications.
+
+With explicit outbox writes, you maintain complete control over which data changes trigger publishes, enabling you to synchronize only the changes required by your application.
+
+You also have total authority over the channels to which change events are published:
+
+* You can distribute change notifications across various "channels":/channels according to the requirements of your application.
+* Channels provide a mechanism for data access control via channel "capabilities":/auth/capabilities.
+
+h3(#schema). Schema
+
+The following table describes the schema of the outbox table in your database:
+
+|_. Field |_. Type |_. Required on write? |_. Example Value |_. Description |
+| @sequence_id@ | Serial integer (primary key) | No (automatically assigned) | @1@ | Monotonically increasing identifier that determines publish order within the scope of the channel. |
+| @mutation_id@ | String | Yes | @680f3f78-21ec-49a0-be99-25f89a84f232@ | TThe ID of the mutation, used for correlating the outbox event with an optimistic event applied locally on the client. |
```

Comment:
Typo on `TThe`

Reply:
Fixed in this [commit](https://github.com/ably/docs/pull/2082/commits/15a2be26866049471006c24c9e1e9094c5b53a85)